packages/jaeger-ui/src/components/TracePage/TraceStatistics/index.tsx

```diff
@@ -392,79 +263,168 @@ export default class TraceStatistics extends Component<Props, State> {
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'Count',
+        dataIndex: 'count',
+        sorter: (a,b) => sorterFunction({"type": a.type,"index": a.count},{"type": b.type,"index": b.count}),
+        onCell: (record) => onCellFunction(record),
+        defaultSortOrder: 'ascend'
+      },
+      {
+        title: 'Total',
+        dataIndex: 'total',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.total},{"type": b.type,"index": b.total}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'Avg',
+        dataIndex: 'avg',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.avg},{"type": b.type,"index": b.avg}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'Min',
+        dataIndex: 'min',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.min},{"type": b.type,"index": b.min}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'Max',
+        dataIndex: 'max',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.max},{"type": b.type,"index": b.max}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Total',
+        dataIndex: 'selfTotal',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfTotal},{"type": b.type,"index": b.selfTotal}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Avg',
+        dataIndex: 'selfAvg',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfAvg},{"type": b.type,"index": b.selfAvg}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Min',
+        dataIndex: 'selfMin',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfMin},{"type": b.type,"index": b.selfMin}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Max',
+        dataIndex: 'selfMax',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfMax},{"type": b.type,"index": b.selfMax}),
+        render: (cell: string) => {
+              return cell+'ms'
```

Comment:
```suggestion
              return `${cell} ms`
```

You can switch the other string concatenation with the above format.